src/Pages/ChannelRequests.razor

```diff
@@ -596,7 +596,7 @@
             {
                 _manageableNodes = _manageableNodes.Where(node => node.Id != _selectedDestNode.Id).ToList();
             }
-            _allWallets = await WalletRepository.GetAvailableWallets();
+            _allWallets = await WalletRepository.GetAvailableWallets(false);
```

Comment:
Is it possible to send the funds when closing a channel to a watch-only wallet?

Reply:
Nope, as they are not able to be used for opening channels.